src/main/java/ubc/pavlab/rdp/model/ontology/OntologyTermInfo.java

```diff
@@ -0,0 +1,152 @@
+package ubc.pavlab.rdp.model.ontology;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.*;
+
+/**
+ * Unassociated ontology term.
+ * <p>
+ * If you need a user-associated term, see {@link UserOntologyTerm}.
+ * <p>
+ * This class implements {@link Comparable} so that it can be held {@link Ontology#getTerms()}'s sorted set.
+ *
+ * @author poirigui
+ */
+
+@Entity
+@Table(name = "ontology_term_info",
+        uniqueConstraints = { @UniqueConstraint(columnNames = { "ontology_id", "term_id" }) },
+        indexes = { @Index(columnList = "name") })
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString(of = { "id" }, callSuper = true)
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
```

Comment:
Is this caching annotation justified? And why not just READ_WRITE? (I think I noticed performance issue, but it would be nice to review it again).